src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -32,7 +32,7 @@ public DeploymentOptionProperty(T value, string description = "", string display
         public string Description { get; set; }
 
         // To display the options in ADS extensions UI in SchemaCompare/SQL-DB-Project/Dacpac extensions
-        public string DisplayName { get; set; }
+        public string PropertyName { get; set; }
```

Comment:
why is this getting changed from DisplayName to PropertyName?

Reply:
what's the difference between the display name and property name? Can you give an example?

Reply:
DisplayName: "Restore sequence value" coming from the DacFx_Property_FriendlyName and
PropertName is nothing but a property "RestoreSequenceValue".

Reply:
ok and the PropertyName is what's used to convert from the map value back to the DacDeployOption?

Reply:
Is there a reason DisplayName was selected for the key rather than PropertyName?